### PR TITLE
CI for estuary

### PR DESCRIPTION
**Pull request by dunson062786**
…ockerfile to Estuary project.

### CODE REVIEW

File: apps/estuary/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM smartcitiesdata:build as builder
+RUN MIX_ENV=prod mix distillery.release --name estuary
```
Comment (jmorris-pillar) [inline apps/estuary/Dockerfile]: looks like distillery is not a depdenency in this app yet.

Comment (jmorris-pillar) [reply apps/estuary/Dockerfile]: sorry to give you peacemeal changes.

File: .travis.yml [hunk 2]
```diff
@@ -48,4 +48,5 @@ jobs:
 branches:
   only:
   - master
+  - estuary_umbrella
```
Comment (ManApart) [inline .travis.yml]: This doesn't seem consistent with how we've done the other apps. Are we using this just temporarily and then planning on removing him once we're no longer branching from a branch?

File: apps/estuary/lib/estuary/helper/kafka_helper.ex [hunk 3]
```diff
@@ -0,0 +1,5 @@
+defmodule Estuary.KafkaHelper do
+  def list_exisiting_topics do
+    Elsa.list(localhost: 9092)
```
Comment (ManApart) [inline apps/estuary/lib/estuary/helper/kafka_helper.ex]: May be worth doing an IO.inspect on this guy to see the output when you call this (I'm assuming little test) function.

Comment (dunson062786) [reply apps/estuary/lib/estuary/helper/kafka_helper.ex]: yes

**Comment by kcarra** [comment]
@jmorris-pillar @ManApart Since we've decided to go with 1.9 should we be looking to replace Distillery with `mix release`?

File: apps/estuary/mix.lock [hunk 4]
```diff
@@ -1,12 +0,0 @@
-%{
```
Comment (ManApart) [inline apps/estuary/mix.lock]: Why was this guy deleted?

Comment (dunson062786) [reply apps/estuary/mix.lock]: This is not required for each app. 
-Sukaant

Comment (ManApart) [reply apps/estuary/mix.lock]: Ok, I missed that. Thanks.

File: .travis.yml (hunk 2 above)
Comment (dunson062786) [reply .travis.yml]: yes

**Pull request merged**